docs/user-guide/templates.md

```diff
@@ -0,0 +1,64 @@
+# Templates
+
+Templates are snippets of predefined code that people can use to replace a job definition in a [screwdriver.yaml](./configuration/index.md).
```

Comment:
I think it will be more clear. When we add some other templates then we can come back and add this.